src/Microsoft.Health.Fhir.Liquid.Converter/Ccda/CcdaDataParser.cs

```diff
@@ -49,7 +49,7 @@ public static class CcdaDataParser
                 // Remove line breaks in original data
                 dataDictionary["_originalData"] = Regex.Replace(document, @"\r\n?|\n", string.Empty);
 
-                return new Dictionary<string, object>()
+                return new Dictionary<string, object>
                 {
                     { "msg", dataDictionary },
```

Comment:
There's no special reason for it and the names are originally taken from handlebars templates (`msg.v2`, `msg`). I totally agree that it would be better to use the same keys for Hl7 v2 and C-CDA, but for legacy reasons I recommend that we keep the old names, as they've already used in templates.